Mathlib/Analysis/Asymptotics/Asymptotics.lean

```diff
@@ -2137,6 +2137,10 @@ theorem IsLittleO.right_isBigO_add {f₁ f₂ : α → E'} (h : f₁ =o[l] f₂)
   ((h.def' one_half_pos).right_le_add_of_lt_one one_half_lt_one).isBigO
 #align asymptotics.is_o.right_is_O_add Asymptotics.IsLittleO.right_isBigO_add
 
+theorem IsLittleO.right_isBigO_add' {f₁ f₂ : α → E'} (h : f₁ =o[l] f₂) :
+    f₂ =O[l] (f₂ + f₁) :=
+  Trans.trans h.right_isBigO_add (add_comm _ _)
```

Comment:
Does this work? Untested...
```suggestion
  h.right_isBigO_add.trans (add_comm _ _)
```

Reply:
Unfortunately not, since this is `IsBigO` trans ` Eq`. There is also no `IsBigO.trans_eq` lemma; instead I believe `Trans.trans` is using this instance: https://github.com/leanprover/lean4/blob/56d703db8eec777b813a198e2126115bda9d3a9b/src/Init/Prelude.lean#L1152-L1153